dev/com.ibm.ws.jpa.container.v31/bnd.bnd

```diff
@@ -29,16 +24,13 @@ Private-Package: \
   com.ibm.ws.jpa.container.v31.JPAPCtxtAttributeAccessorV31, \
   com.ibm.ws.jpa.container.v31.internal.JPA31Runtime
 
-javac.source: 11
-javac.target: 11
```

Comment:
Not sure why you removed these.  It should be source and target 11.  It should have these settings:
```
javac.source: 11
javac.target: 11

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
```